server/models/Profile.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
```

Comment:
If you don't want this to show in the diff, you can set the base branch for the PR to your Profile Model branch. Just hit edit at the top of the PR and select a base branch from the drop down

Reply:
Thanks for the tip! I have gone ahead and done so.